components/MarkdownTextBlock/src/MarkdownConfig.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace CommunityToolkit.Labs.WinUI.MarkdownTextBlock;
+
+public record MarkdownConfig
+{
+    public string? BaseUrl { get; set; }
+    public string? Markdown { get; set; }
```

Comment:
Would be good for the text to be its own `Text` dependency property on the control vs. backed into the config, as it may change more than the rest of these things which could be shared between multiple instances of the control.

Reply:
Addressed in the latest version